modules/cnf-image-based-upgrade-share-container-directory.adoc

```diff
@@ -2,15 +2,15 @@
 // * scalability_and_performance/ztp-image-based-upgrade.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="ztp-image-based-upgrade-shared-container-directory_{context}"]
+[id="cnf-image-based-upgrade-shared-container-directory_{context}"]
 = Sharing the container directory between `ostree` stateroots
 
 You must apply a `MachineConfig` to both the seed and the target clusters during installation time to create a separate partition and share the `/var/lib/containers` directory between the two `ostree` stateroots that will be used during the upgrade process.
 
-[id="ztp-image-based-upgrade-shared-container-directory-acm_{context}"]
-== Sharing the container directory between `ostree` stateroots when using {rh-rhacm}
+[id="cnf-image-based-upgrade-shared-container-directory-acm_{context}"]
+== Sharing the container directory between `ostree` stateroots
 
-When you are using {rh-rhacm}, you must apply a `MachineConfig` to both the seed and target clusters.
+You do the following procedure to create a separate disk partition on both the seed and target cluster and to share the `/var/lib/containers` directory.
 
 [IMPORTANT]
```

Comment:
So we apply this `MachineConfig` at install time? In this case, Logging in as a user with `cluster-admin` privs doesn't make sense. We need to apply this as we provision the clusters in the first place?